index.html

```diff
@@ -2,6 +2,10 @@
 <html>
 <head>
 <link href="css/style.css" rel="stylesheet" type="text/css"/>
+<script src="https://cdn.ravenjs.com/3.19.1/raven.min.js" crossorigin="anonymous"></script>
+	<script>
+        Raven.config('https://2e2f7f94e86c471189b46bc12abcf6c9@sentry.pchat.palaceworld.net/2').install()
```

Comment:
It's already being installed in `main.js`, which is the better place, IMO. Also, you're adding some useful tags there.